consumergroup/consumer_group.go

```diff
@@ -77,6 +77,7 @@ type ConsumerGroup struct {
 	instance *kazoo.ConsumergroupInstance
 
 	wg             sync.WaitGroup
+	nGoroutines    int
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**suggestion (non-blocking):** ‏`nGoroutines` was added while debugging, as it does not bring any additional value we can remove it now.